src/Adapter/MSTest.TestAdapter/SourceGeneration/SourceGeneratedReflectionOperations.cs

```diff
@@ -84,7 +84,7 @@ public MethodInfo[] GetRuntimeMethods(Type type)
         Dictionary<string, PropertyInfo> type = ReflectionDataProvider.TypePropertiesByName[classType];
 
         // We as asking for TestContext here, it may not be there.
-        PropertyInfo? property = type.TryGetValue(propertyName, out PropertyInfo? propertyInfo) ? propertyInfo : null;
+        PropertyInfo? property = type.GetValueOrDefault(propertyName);
```

Comment:
```suggestion
        _ = type.TryGetValue(propertyName, PropertyInfo? property);
```

Reply:
They are the same, nothing really wrong about `GetValueOrDefault`.

`GetValueOrDefault` will end up calling TryGetValue at the end of the day, so I preferred to call TryGetValue directly unless `GetValueOrDefault` is saving annoying conditional syntax (like the case in `GetMetadata` below in the PR)

It's mostly personal opinion though.

Reply:
give my preference for this style does not match the projects's, i will close